Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -112,7 +117,7 @@ private void Scrobble(PlexEvent data)
         #endregion
 
         [NonAction]
-        private static (SVR_AnimeEpisode, SVR_AnimeSeries) GetEpisode(PlexEvent.PlexMetadata metadata)
+        private (SVR_AnimeEpisode, SVR_AnimeSeries) GetEpisodeAndSeries(PlexEvent.PlexMetadata metadata)
```

Comment:
There was no need to change this from static.

Reply:
It was needed since I switch the logger from a static logger to an dynamically injected logger.